app/views/dashboard/_buyer.html.erb

```diff
@@ -1,15 +1,45 @@
-<h1 style="margin-top: 120px; text-align:center;">Purchased Properties</h1>
 <% props = Property.where(buyer_id: current_user.id) %>
```

Comment:
View is only for presenting the data, Not for fetching or collecting the data. So please remove this line from here.

Reply:
done